README.md

```diff
@@ -120,11 +126,24 @@ Usage
 
 Running sccache is like running ccache: wrap your compilation commands with it, like so:
 
-> $ sccache gcc -o foo.o -c foo.c
+```bash
+sccache gcc -o foo.o -c foo.c
+```
+
+If you want to use sccache for your rust builds you can define `build.rustc-wrapper` in the
+[cargo configuration file](https://doc.rust-lang.org/cargo/reference/config.html).  For example, you can set it globally
+in `$HOME/.cargo/config` by adding:
 
-or use it with rust, like so:
+```toml
+[build]
+rustc-wrapper = "[path to sccache]"
+```
+
+Alternatively you can use the environment variable `RUSTC_WRAPPER`:
 
-> $ RUSTC_WRAPPER=[path to sccache] cargo build
+```text
+RUSTC_WRAPPER=[path to sccache] cargo build
+```
```

Comment:
I think you are inconsistent about whether `text` or `bash` is used for commands (or not used at all, see the `scoop` instructions above).  I don't know what the right choice is, but we should pick one and stick with it.

Reply:
I did that because in these two places because it is actually not an executable bash command, since it includes the placeholder `[path to sccache]`.  In any case it doesn't matter much.  I will switch to `bash`.